docs/data-management/post-session.md

```diff
@@ -374,11 +374,78 @@ ses-01
+
+
+    ??? tip "Installing DataLad"
+        If datalad is not installed on the HPC, the most convenient user-based installation can be achieved by using Conda.
+
+        !!! warning "Do not install DataLad on a login node"
+            HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing jobscripts.
+            Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
+            Interactive sessions are a great alternative when available and **should** be used when creating the DataLad dataset.
+            For example, in the case of systems operating SLURM, the following command would open a new interactive session:
+            ``` bash
+            srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
+            ```
+
+        - [ ] Start an interactive session on the HPC server
+
+        ??? tip "Installing with `pip`"
+            - [ ] Load python if it is not available (here we use `module`):
+                - [ ] Check the dependencies for your specific python version (here `3.8.2`):
+                ``` bash
+                module spider Python/3.8.2
+                ```
+                - [ ] Load python along with its dependencies (`ml` is equivalent to `module load`)
+                ``` bash
+                ml GCCcore/9.3.0 Python/3.8.2
+                ```
+            - [ ] Now that `pip` is installed, install DataLad:
+                ``` bash
+                pip install --user datalad
+                ```
+        
+        ???+ tip "Installing with `conda`"
+            - [ ] Install some conda distribution (in this example we show *Miniconda*):
+                ``` bash
+                wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+                bash Miniconda3-latest-Linux-x86_64.sh
+                ```
+            - [ ] Install DataLad:
+                ``` bash
+                conda install -c conda-forge -y "datalad>=0.19.0"
+                conda install -c conda-forge datalad-container
+                ```
+        - [ ] Finally, don't forget to configure the Git identity.
+            ``` bash
+            git config --global --add user.name "Bob McBobFace"
+            git config --global --add user.email bob@example.com
+            ```
+        
+    - [ ] Go in the repository where you usually store datasets and create an empty DataLad dataset by running:
+        ```
+        cd /data/datasets/
+        datalad create -c yoda "hcph"
+        ```
+        Using the `yoda` procedure builds the infrastructure of your dataset so it is compliant to the [Yoda principles](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html).
```

Comment:
Looks good to me. I am just wondering if this part should also be done while in an interactive session of the HPC. If it is the case, I would suggest to explicitly say it:
```suggestion
    - [ ] While in an interactive session of the HPC, go in the repository where you usually store datasets and create an empty DataLad dataset by running:
        ```
        cd /data/datasets/
        datalad create -c yoda "hcph"
        ```
        Using the `yoda` procedure builds the infrastructure of your dataset so it is compliant to the [Yoda principles](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html).
```

Reply:
@acionca I think this is not critical. `datalad create` is a lightweight operation that just creates a folder basically